repos/system_upgrade/el7toel8/libraries/utils.py

```diff
@@ -1,11 +1,36 @@
 import functools
 import sys
 
+import six
+
 from leapp.exceptions import StopActorExecutionError
 from leapp.libraries.common import mounting
 from leapp.libraries.stdlib import STDOUT, CalledProcessError, api, config, run
 
 
+def parse_config(cfg):
+    """
+    Applies a workaround using ConfigParser in Python2 or Python3.
```

Comment:
wrong doc. Applying workround is irrelevant msg. Write what the function is supposed to do, what is the argument..

Reply:
Solved on the next commit.